apps/reference/docs/guides/database/extensions/pgnet.mdx

```diff
@@ -0,0 +1,254 @@
+---
+id: pgnet
+title: 'pg_net: Async Networking'
+description: 'pg_net: an async networking extension for PostgreSQL.'
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+:::caution
+
+The pg_net API is not at 1.0 (stable). Functions signatures may change.
```

Comment:
Maybe this should say it's still alpha/beta instead so it sounds a bit less negative.